packages/react-scripts/config/env.js

```diff
@@ -8,10 +8,10 @@
  * of patent rights can be found in the PATENTS file in the same directory.
  */
 // @remove-on-eject-end
+'use-strict';
```

Comment:
We need to figure out why linter doesn't catch it. I specifically changed our internal linter settings yesterday to force `'use strict'`.